frontend/src/helpers/sorting.ts

```diff
@@ -61,6 +62,16 @@ export function getSortGroupUsersFunc(
       return b_value - a_value
     }
 
+    if (a.permissions.length !== b.permissions.length) {
+      return b.permissions.length - a.permissions.length
+    }
+
+    for (let i = 0; i < a.permissions.length; i++) {
+      if (a.permissions[i] !== b.permissions[i]) {
+        return isAtLeastAsValuableRole(a.permissions[i], b.permissions[i]) ? 1 : -1
```

Comment:
`isAtLeastAsValuableRole` has been removed. One solution would be to get the roles index from the roles field included in the GET /group request and then compare those.